Code/Framework/AzFramework/Platform/Linux/AzFramework/Input/LibEVDevWrapper.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Debug/Trace.h>
+#include <AzFramework/Input/LibEVDevWrapper.h>
+
+namespace AzFramework
+{
+    LibEVDevWrapper::LibEVDevWrapper()
+    {
+        m_libevdevHandle = AZ::DynamicModuleHandle::Create("libevdev.so");
+        if (!m_libevdevHandle->Load(false))
+        {
+            m_libevdevHandle.reset();
```

Comment:
Maybe a AZ_Warning here informing that the package for `libevdev` was not found and needs to be installed?

Reply:
Since this isn't an actual warning or error, I've moved it to info, so it will show up in the logs and recommend that they install libevdev, but won't otherwise be a problem...